zerver/lib/message.py

```diff
@@ -408,13 +429,25 @@ def has_message_access(
         # You can't access messages in deactivated streams
         return False
 
-    def is_subscribed_helper() -> bool:
-        if is_subscribed is not None:
-            return is_subscribed
+    def has_content_access_helper() -> bool:
```

Comment:
`has_channel_content_access_helper` might be a bit clearer, just because someone new reading this in the abstract might be thinking about "content access to messages", which isn't a thing.